source/Editor.js

```diff
@@ -125,17 +125,22 @@ function Squire ( root, config ) {
     this._path = '';
     this._willUpdatePath = false;
 
-    if ( 'onselectionchange' in doc ) {
-      this.addEventListener( 'selectionchange', this._updatePathOnEvent );
-    } else {
+    // nodes.  To do this with the 'selectionchange' event is trickier since our current procedure triggers
```

Comment:
in the mainline branch there was a move towards selectionchange for setting ranges, and this interferred with our positioning whenever the cursor was in a not-editable node, so I reverted back to the old behavior